shell_godownloader.go

```diff
@@ -89,6 +89,7 @@ execute() {
     install "${srcdir}/${binexe}" "${BINDIR}/"
     log_info "installed ${BINDIR}/${binexe}"
   done
+  rm -rf "${tmpdir}"
```

Comment:
Maybe its safer if we use a `trap`?

Reply:
I believe the different downloader 'variants' like raw https://github.com/goreleaser/godownloader/blob/master/shell_raw.go#L108 may need a similar application

Reply:
sounds good, thanks!